### ISSUE TITLE
The cost definition for SASD 

### ISSUE THREAD

**Issue by RhythmOfTheRain-Byte** [CVMI-Lab/ST3D#80]
Excuse me, May i ask you where is the definition of the cost function for SASD， i cannot find it in the file of “pcdet/models/detectors/second_net_iou.py”  
----------------------------------------------------------
    def forward(self, batch_dict):
        batch_dict['dataset_cfg'] = self.dataset.dataset_cfg
        for cur_module in self.module_list:
            batch_dict = cur_module(batch_dict)

        if self.training:
            weights = batch_dict.get('SEP_LOSS_WEIGHTS', None)
            loss, tb_dict, disp_dict = self.get_training_loss(weights)

            ret_dict = {
                'loss': loss
            }
            return ret_dict, tb_dict, disp_dict
        else:
            pred_dicts, recall_dicts = self.post_processing_multicriterion(batch_dict)
            return pred_dicts, recall_dicts
-----------------------------------------------------------
When the fellowing part is in the file of “tools/cfgs/da-nuscenes-kitti_models/secondiou_st3d/secondiou_st3d++_ros_car.yaml” 
SELF_TRAIN:
    SRC:
        USE_DATA: True
        USE_GRAD: True
        LOSS_WEIGHT: 1.0
        SEP_LOSS_WEIGHTS: [0.0,1.0,0.0]
    TAR:
        USE_DATA: True
        LOSS_WEIGHT: 1.0
        SEP_LOSS_WEIGHTS: [1.0,1.0,1.0]

Thank you very much！

**Comment by jihanyang** [comment]
https://github.com/CVMI-Lab/ST3D/blob/2634561684dfbafce6ed86e50c6f70f51988593c/tools/train_utils/train_st_utils.py#L44-L61
